config.template.ini

```diff
@@ -50,3 +50,8 @@ FEED_CHECK_DEPTH = 2500
 ; Constants
 [api]
 MAX_UNDERRATED_CUSTOM_POPULARITY=0.5
+
+; Scheduler
+[scheduler]
+FB_FETCH_FREQUENCY = 5
+PROCESS_POSTS_FREQUENCY = 5
```

Comment:
Every 15 minutes should be sufficient. 5 minute mein toh ek gaana khatam nai hota hai.

Reply:
Besides, our frontend won't fetch new posts until someone refreshes too. We'll change it back to 5 if needed in future. There isn't really any resources being used here, but let's give it a week to understand any possible issues.

Reply:
Yeah, I just set this during testing, I'll change it right now.